### PR TITLE
glbinding: allow building static libraries

### PR DESCRIPTION
**Pull request by rggjan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/glbinding.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Glbinding < Formula
   homepage "https://github.com/cginternals/glbinding"
   url "https://github.com/cginternals/glbinding/archive/v2.1.3.tar.gz"
   sha256 "48f2e590a4a951005f79fec6c487217aa9b344a33ca1a8d2b7e618f04681ec60"
+  revision 1
```
Comment (ilovezfs) [inline Formula/glbinding.rb]: no revision is needed for a non-default option

Comment (rggjan) [inline Formula/glbinding.rb]: Thanks! Fixed!

**Comment by JCount** [comment]
@rggjan Thank you for your contribution! 🎆

**Comment by rggjan** [comment]
You're welcome :)

**Pull request merged**